### PR TITLE
add basic web app using flask

### PR DESCRIPTION
**Pull request by Roboto0**

### CODE REVIEW

File: stockAndPrices/services/app/README.MD [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Python version : 3.79
+
+"pip install flask" in command line to download flask
+
+
```
Comment (allenaerostar) [inline stockAndPrices/services/app/README.MD:5]: Leave a comment on how to start up flask server

File: stockAndPrices/services/app/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Home Page</title>
+    </head>
+
+    <body>
```
Comment (allenaerostar) [inline stockAndPrices/services/app/templates/index.html:7]: We can leave these html files in for now, but remove them once the react project is hooked up

File: stockAndPrices/services/app/README.MD (hunk 1 above)
Comment (Roboto0) [reply stockAndPrices/services/app/README.MD:25]: Updated Readme on how to start the server!

File: stockAndPrices/services/app/templates/index.html (hunk 2 above)
Comment (Roboto0) [reply stockAndPrices/services/app/templates/index.html]: Removed the two html files and the templates folder.

**Review by allenaerostar — APPROVED** [review]

**Review by lilianahechen — APPROVED** [review]

**Pull request merged**